Portfolio/src/assets/Variables.scss

```diff
@@ -1,11 +1,23 @@
-// $color-pink-primary: #D4ABB2;
-// $color-black-heading: #1e1e1e;
-// $color-white: #ffff;
-
-// $font-family-primary: "Raleway";
-// $font-size-small: 1.5rem;
-
-// $margin-top-small: 5rem;
-// $margin-top-large: 10rem;
+$margin-top-xsmall: 1rem;
+$margin-top-medium: 5rem;
+$margin-top-large: 10rem;
+$margin-zero: 0;
+$margin-medium: 4rem;
+$padding-small: 10px;
+$width-small: 2rem;
+$width-xlarge: 15rem;
 
+// Color Variables
+$color-pink-primary: #d4abb2;
+$color-black: #1e1e1e;
+$color-white: #ffff;
 
+// Font Variables
+$font-family-primary: 'Raleway';
+$font-size-xsmall: 1.5rem;
+$font-size-small: 2rem;
+$font-size-medium: 3rem;
+$font-size-large: 4rem;
+$font-size-xlarge: 5rem;
+$letter-spacing: -2px;
+$text-transform-uppercase: uppercase;
```

Comment:
Tbh this variable is not really needed in my eyes, as using this variable is the same as saying: `text-transform: uppercase`.  I dont think u need a variable for it. Variables are there to have consistency and write less error prone and more organied code, and in this case it does not add any value 🌷